lib/virt_feature_test_base.pm

```diff
@@ -157,7 +158,21 @@ sub post_fail_hook {
     my ($self) = shift;
 
     $self->{"stop_run"} = time();
-    assert_script_run("history -w /root/commands_history");
+    unless (defined(script_run("history -w /root/commands_history", die_on_timeout => 0))) {
```

Comment:
What if 
```
script_run("history -w /root/commands_history", die_on_timeout => 0)
```
returns value greater than 0 ?

Reply:
In the case of timeout, such as broken ssh connection, there is no return value, ie. the return valude is 'undefined'. 'undefined' will result in compile error in a condition statement. other return values I have seen are '0' and '1'.

Reply:
I did not see 'What if' in your comment. 
Here the change handles only the case of ssh command timeout.